CollAction/Controllers/ProjectsController.cs

```diff
@@ -364,12 +365,35 @@ public ViewResult StartInfo()
         }
 
         [HttpGet]
-        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId)
+        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId, string statusId)
         {
-            var displayTileProjectViewModels = await _service.GetTileProjects(Url, p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted
+            Expression<Func<Project, bool>> projectExpression = (p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted
             && (categoryId != null ? p.CategoryId == categoryId : true) && (locationId != null ? p.LocationId == locationId : true));
+
+            Expression<Func<Project, bool>> statusExpression;
+            switch (String.IsNullOrEmpty(statusId) ? String.Empty : statusId.ToLower())
+            {
+                case "open": statusExpression = (p => p.IsActive == true); break;
```

Comment:
You're using a lot of unmapped properties here. It's best not to search for unmapped properties, those don't get optimized to sql. Very inefficient, means the whole project table is selected, and then filtered client-side.

Just search for the implementation, like:
        public bool IsClosed
            => Status == ProjectStatus.Running && End < DateTime.UtcNow;